Rakefile

```diff
@@ -218,13 +218,13 @@ file presym_inc => presym_file do
         f.print "MRB_PRESYM_CSYM(#{sym}, #{i+1})\n"
       elsif op_table.key?(sym)
         f.print "MRB_PRESYM_QSYM(\"#{sym}\", #{op_table[sym]}, #{i+1})\n"
-      elsif /\?\Z/ =~ sym
+      elsif /\A\w+\?\Z/ =~ sym
```

Comment:
It should be `[A-Za-z_]\w*` instead of `\w+` (`[A-Za-z0-9_]+`).

Reply:
Sure, fixed!